anthos-multi-cloud/README.md

```diff
@@ -7,3 +7,7 @@ Checkout the different folders here for a Terraform based quickstart for setting
 - [Run GKE on AWS](/anthos-multi-cloud/AWS)
 - [Run GKE on Azure](/anthos-multi-cloud/Azure)
 
+The `customize-logs-fluentd` directory contains files to deploy a Fluent Bit
+DaemonSet on an Anthos Multi-Cloud cluster. Fluent Bit can be used to
+customize Cloud Logging logs for Anthos clusters on AWS and Anthos clusters
+on Azure.
```

Comment:
Can we add this as a sub item to the list?

- [customize-logs-fluentd](/anthos-multi-cloud/customize-logs-fluentd): This directory contains files to deploy a Fluent Bit DaemonSet on an Anthos Multi-Cloud cluster. Fluent Bit can be used to customize Cloud Logging logs for Anthos clusters on AWS and Anthos clusters on Azure.